package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "eslint": "eslint --ignore-path .gitignore .",
     "eslint-fix": "eslint --fix --ignore-path .gitignore .",
-    "test": "npm run eslint",
+    "test": "jest",
```

Comment:
Leave the `test` script alone, and instead add one for `jest`:

```json
"test": "npm run eslint",
"jest": "jest",
```

In the new bug you file we can combine these via a call to `"pretest": "npm run eslint"` and change `test` to run `jest`

Reply:
Fixed in https://github.com/Seneca-CDOT/telescope/commit/c3d8a6a55d1ba26c4180f3a2ac7e48247062f97c.